src/AuthContextInterface.ts

```diff
@@ -136,7 +136,7 @@ export interface AuthContextProps {
   /**
    *
    */
-  signOutRedirect: (args?: unknown) => Promise<void>;
+  signOutRedirect: (args?: SignoutRedirectArgs) => Promise<void>;
```

Comment:
This might brake some installs; we should probably bump the major version. (easily just add [`!` after type of change](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change)).

Reply:
Is there a suggested means of bumping the major version after the fact?

Reply:
Our workflows should figure that out automagically. :)